lib/DDG/Spice/LaunchLibrary.pm

```diff
@@ -0,0 +1,31 @@
+package DDG::Spice::LaunchLibrary;
+# ABSTRACT: List upcoming rocket launches and where to watch them online
+
+use strict;
+use DDG::Spice;
+
+spice is_cached => 1;
+
+name "Launch Library";
+source "LaunchLibrary.net";
+description "List upcoming rocket launches and where to watch them online";
+primary_example_queries "upcoming rocket launches", "next space launch";
+category "reference";
+topics "geek", "science";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/LaunchLibrary.pm";
+attribution github => ["globalspin", "Chris Radcliff"],
+            twitter => "chris_radcliff";
+
+# Version 1.1 of the API allows GET requests. 
+# Parameters are included in the URL path; "/next/5" shows the next 5 launches
+# See launchlibrary.net for the API reference.
+spice to => 'http://launchlibrary.net/1.1/launch/next/5';
+spice wrap_jsonp_callback => 1;
+
+triggers any => "rocket launches", "rocket launch", "space launches", "space launch";
+
+handle remainder => sub {
+    return 1;
```

Comment:
I think it would be best to `handle query_lc` and use a regex to make sure the query fit's the form we expect.

Right now it will trigger and show this IA for any and all queries that have "rocket launch" anywhere in the query.

Looking at the autocomplete suggestions in the browser, you can see almost every query that starts with "rocket launch" isn't one that should trigger this Instant Answer:

<img width="736" alt="drake_at_duckduckgo" src="https://cloud.githubusercontent.com/assets/873785/9610656/037a637a-50a9-11e5-8bca-49a9b5a25d57.png">